diff -rupN ubase-20181115/passwd.c ubase.new/passwd.c
--- ubase-20181115/passwd.c	2018-11-15 22:28:47.000000000 +0100
+++ ubase.new/passwd.c	2018-12-07 00:37:11.000000000 +0100
@@ -235,11 +235,16 @@ newpass:
 		eprintf("getpass:");
 	if (inpass[0] == '\0')
 		eprintf("no password supplied\n");
-	p = crypt(inpass, prevhash);
-	if (!p)
+		
+	/* compare old password if requested */
+	if (cryptpass1) {
+	    p = crypt(inpass, prevhash);
+	    if (!p)
 		eprintf("crypt:");
-	if (cryptpass1 && strcmp(cryptpass1, p) == 0)
+	    if (strcmp(cryptpass1, p) == 0)
 		eprintf("password left unchanged\n");
+	}
+	
 	gensalt(salt + strlen(salt));
 	p = crypt(inpass, salt);
 	if (!p)
