diff -rupN svc-20181204/bin/svc svc.new/bin/svc
--- svc-20181204/bin/svc	2018-12-04 07:32:35.315730075 +0100
+++ svc.new/bin/svc	2018-12-04 14:51:39.674885906 +0100
@@ -3,7 +3,7 @@
 # See LICENSE file for license details.
 #
 
-BASEDIR="/bin/svc.d"
+BASEDIR="/etc/rc.d"
 BASENAME="$(basename $0)"
 
 [ -d "$BASEDIR" ] || mkdir -p "$BASEDIR"
@@ -26,9 +26,9 @@ runservice() {
 
 	if [ $(stat -L -t "$dir/$service" | cut -d' ' -f 2) -eq 0 ];
 	then
-		"./bare.sh" "$service" "$cmd" 2>&1 >/dev/null &
+		"./bare.sh" "$service" "$cmd" 2>&1 >/dev/null
 	else
-		"$dir/$service" "$cmd" 2>&1 >/dev/null &
+		"$dir/$service" "$cmd" 2>&1 >/dev/null
 	fi
 }
 
diff -rupN svc-20181204/svc.d/bare.sh svc.new/svc.d/bare.sh
--- svc-20181204/svc.d/bare.sh	2018-12-04 07:32:35.316730063 +0100
+++ svc.new/svc.d/bare.sh	2018-12-04 14:49:10.457751377 +0100
@@ -8,7 +8,7 @@ then
 	shift 1
 fi
 
-[ -e "/bin/svc.d/default/$SERVICE" ] && . "/bin/svc.d/default/$SERVICE"
+[ -e "/etc/rc.d/default/$SERVICE" ] && . "/etc/rc.d/default/$SERVICE"
 
 BIN=""
 for p in /bin /sbin /usr/bin /usr/sbin /usr/local/bin /usr/local/sbin;
@@ -21,7 +21,7 @@ do
 done
 [ -z "$BIN" ] && exit 1
 
-PID=$(pidof -o %PPID $BIN)
+PID=$(pidof -o %PPID $BIN 2> /dev/null)
 
 case $1 in
 	-s)
