From 6a34b7e6505870697e2acf351438dc6affe6b3a2 Mon Sep 17 00:00:00 2001
From: Jason Woodward <woodwardj@jaos.org>
Date: Sun, 11 Jul 2021 14:43:10 -0400
Subject: [PATCH] move actions into main (2): update legacy Makefile

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fd1b5dc..f663b2b 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ CURLFLAGS=`curl-config --libs` -lcrypto
 OBJS=src/common.o src/configuration.o src/package.o src/slaptcurl.o src/transaction.o src/action.o src/main.o
 LIBOBJS=src/common.o src/configuration.o src/package.o src/slaptcurl.o src/transaction.o
 LIBHEADERS=src/main.h src/common.h src/configuration.h src/package.h src/slaptcurl.h src/transaction.h
-NONLIBOBJS=src/action.o src/main.o
+NONLIBOBJS=src/main.o
 RCDEST=/etc/slapt-get/slapt-getrc
 RCSOURCE=default.slapt-getrc.$(ARCH)
 PACKAGE_LOCALE_DIR=/usr/share/locale

