diff -rupN minidlna-1.3.0/configure.ac minidlna-1.3.0-new/configure.ac
--- minidlna-1.3.0/configure.ac	2020-11-24 19:53:50.000000000 +0100
+++ minidlna-1.3.0-new/configure.ac	2020-12-23 19:01:58.994103664 +0100
@@ -486,9 +486,9 @@ AC_CHECK_LIB(avahi-client, avahi_threade
 ################################################################################################################
 ### Header checks
 
-AC_CHECK_HEADERS([arpa/inet.h asm/unistd.h endian.h machine/endian.h fcntl.h libintl.h locale.h netdb.h netinet/in.h poll.h stddef.h stdlib.h string.h sys/file.h sys/inotify.h sys/ioctl.h sys/param.h sys/queue.h sys/socket.h sys/time.h unistd.h])
+AC_CHECK_HEADERS([arpa/inet.h asm/unistd.h endian.h machine/endian.h fcntl.h libintl.h locale.h netdb.h netinet/in.h poll.h stddef.h stdlib.h string.h sys/file.h sys/inotify.h sys/ioctl.h sys/param.h bsd/sys/queue.h sys/socket.h sys/time.h unistd.h])
 test x"$ac_cv_header_poll_h" != x"yes" && AC_MSG_ERROR([poll.h not found or not usable])
-test x"$ac_cv_header_sys_queue_h" != x"yes" && AC_MSG_ERROR([sys/queue.h not found or not usable])
+test x"$ac_cv_header_bsd_sys_queue_h" != x"yes" && AC_MSG_ERROR([bsd/sys/queue.h not found or not usable])
 
 AC_CHECK_FUNCS(inotify_init, AC_DEFINE(HAVE_INOTIFY,1,[Whether kernel has inotify support]), [
     AC_MSG_CHECKING([for __NR_inotify_init syscall])
